ndb/tests/unit/test_model.py

```diff
@@ -1370,7 +1370,7 @@ def test_compare_valid():
     def test_compare_invalid():
         prop = model.ModelKey()
         with pytest.raises(exceptions.BadValueError):
-            prop == None
+            prop == None  # noqa: E711
```

Comment:
What is `E711`? Is there a way I can send edits to this PR to avoid using comment for this

Reply:
It thinks this should read `prop is None`.